File: 6) C E C++/calculoRaio.cpp
#include <iostream>

int main( ) {
	float area, raio, calculo;

	std::cout << "Digite o raio do circulo: ";
	std::cin >> raio;
	
	calculo = raio * raio;
	area = 3.14 * calculo;
	
	std::cout << "Area: " << area << std::endl;
}

File: 4) C#/calculoRaio.cs
using System;

namespace CalculoRaio
{
	class Program
	{
		static void Main( string[ ] args )
		{
			double raio, area;
			
			Console.WriteLine( "Digite o raio do circulo: " );
			raio = float.Parse( Console.ReadLine ( ) );
			
			area = 3.14 * ( raio * raio );
			Console.WriteLine( $"Area circulo: {area}" );
		}
	}
}